docs/conan.md

```diff
@@ -0,0 +1,147 @@
+
+## Supported platforms
+
+The following platforms are supported and known to work, others might require changes to our [conanfile.py](../conanfile.py) or upstream recipes.
+
+- **macOS**: x86_64 (Intel) - target 10.13 (High Sierra), arm64 (Apple Silicon) - target 11.0 (Big Sur)
+- **iOS**: arm64 - target 12.0
+
+## Getting started
+
+1. [Install Conan](https://docs.conan.io/en/latest/installation.html)
+2. Execute in terminal: `conan profile new default --detect`
+
+## Download dependencies
+
+0. If your platform is not on the list of supported ones or you don't want to use our prebuilt binaries, you can still build dependencies from source or try consuming prebuilt binaries from the central Conan repository - [ConanCenter](https://conan.io/center/). In this case skip to the next section directly.
+
+1. Check if your build environment can use the prebuilt binaries: basically, that your compiler version (or Xcode major version) matches the information below. If you're unsure, just skip to next step.
+
+    - macOS: libraries are built with Apple clang 13 (Xcode 13.4.1), should be consumable by Xcode and Xcode CLT 13.x
+    - iOS: libraries are built with Apple clang 13 (Xcode 13.4.1), should be consumable by Xcode 13.x
+
+2. Download the binaries archive and unpack it to `~/.conan` directory:
+
+    - [macOS](https://github.com/vcmi/vcmi-deps-macos/releases/latest): pick **intel.txz** if you have Intel Mac, otherwise - **intel-cross-arm.txz**
+    - [iOS](https://github.com/vcmi/vcmi-ios-deps/releases/latest)
+
+3. Only if you have Apple Silicon Mac and trying to build for macOS or iOS: follow [instructions how to build Qt host tools for Apple Silicon](https://github.com/vcmi/vcmi-ios-deps#note-for-arm-macs), on step 3 copy them to `~/.conan/data/qt/5.15.x/_/_/package/SOME_HASH/bin` (`5.15.x` and `SOME_HASH` are placeholders).
+
+## Generate CMake integration
+
+Conan needs to generate CMake toolchain file to make dependencies available to CMake. See `CMakeDeps` and `CMakeToolchain` [in the official documentation](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake.html) for details.
+
+In terminal `cd` to the VCMI source directory and run the following command. If you want to download prebuilt binaries from ConanCenter, also read the next section.
+
+<pre>
+conan install . \
+  --install-folder=<b><i>conan-generated</i></b> \
+  --no-imports \
+  --build=<b><i>never</i></b> \
+  --profile:build=default \
+  --profile:host=<b><i>CI/conan/PROFILE</i></b>
+</pre>
+
+The highlighted parts can be adjusted:
+
+- ***conan-generated***: directory where the generated files will appear. This value is used in CMake presets from VCMI, but you can actually use any directory and override it in your local presets.
```

Comment:
please, specify is it relative path or absolute? Probably should be some recommendation (assuming we and in VCMI sources repo, does it make sense to advice something like ../vcmi-deps?)

Reply:
can be relative or absolute, it's up to you. Also see examples: there're both.